src/App.tsx

```diff
@@ -15,36 +17,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE = '',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState(SortType.NONE);
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```

When value contain `boolean` we start the name with `is`